### DISCUSSION TITLE
The help overlay is not scrollable

### DISCUSSION THREAD

**Discussion by infromthecold** [maziac/DeZog#62] [General]
The help content overlay window is not scrollable.

To recreate 
click DEZOG help on the left
then choose a topic
then click CONTENT

the overlay is not scrollable, so some of the contents is off the screen and inaccessible from that menu

**Comment by maziac** [comment]
Yes, that's true. I was too lazy to implement this :-)

**Comment by maziac** [comment]
Pro-Tip:
You can also use search (CTRL-F) in the help window:
![Bildschirmfoto 2021-08-13 um 17 25 15](https://user-images.githubusercontent.com/3081973/129382174-5d05648d-9f9b-48dd-a6c0-4ecc160942e2.jpg)

**Comment by infromthecold** [comment]
That's great, that will help, as for the expressions, that's a great idea,
depending on how complex the expressions can be.

Sorry but i have my head in my life at the moment, so responses will be
slow

:)
Great stuff though

**Comment by maziac** [comment]
Is scrollable now (2.4).
